posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -320,15 +322,36 @@ metadata:
 spec:
   managementState: "Unmanaged"
 ```
-*Note:* After you set this value to *"Unmanaged"*, any further changes to the _ClusterLogging_ or _LogForwarding_ instances are not automatically detected. To automatically detect changes, you must change the *managementState* field back to *"Managed"*.
 
-Next, set the *MERGE_JSON_LOG* environment variable to *true*.
+Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Administration > Custom Resource Definitions > ClusterLogging > Instances > Instance > YAML`
```

Comment:
it seems like we are telling the user that they can make the change without telling them how.

Alternatively, you can modify this file on the web console. Ensure you are in the `openshift-logging` project namespace and go to `Administration > Custom Resource Definitions > ClusterLogging > Instances > Instance > YAML`. Change the *managementState* field from *"Managed"* to *"Unmanaged"*.

Reply:
fixed